CHANGELOG.md

```diff
@@ -6,7 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/), and this
 
 <!-- ## Unreleased [{version_tag}](https://github.com/opengisch/qgis-plugin-ci/releases/tag/{version_tag}) - YYYY-MM-DD -->
 
-## Unreleased
```

Comment:
Did you remove the `Unreleased` on purpose because we don't support it when parsing for your own version metadata ?

Reply:
No, because it's empty an dduplicated with the comment above.